manifests/backup/xtrabackup.pp

```diff
@@ -47,12 +43,11 @@
     require => Package['percona-xtrabackup'],
   }
 
-  file { 'mysqlbackupdir':
+  file { $backupdir:
```

Comment:
nope, this is a bit over-enthusiastic.
we need this for our dependencies, and our spec tests.